src/stories/html-editor.stories.js

```diff
@@ -3,19 +3,22 @@ import React, { useState } from "react";
+  );
```

Comment:
Did you forget this? It's awesome hahaha

Reply:
This is for Storybook, the playground to test the editor. If you run `npm run storybook` some magic happens.